bin/brew

```diff
@@ -13,6 +16,8 @@ fi
 BREW_FILE_DIRECTORY="$(chdir "${0%/*}" && pwd -P)"
 HOMEBREW_BREW_FILE="$BREW_FILE_DIRECTORY/${0##*/}"
 
+HOMEBREW_PREFIX="$(chdir "$(dirname "$(dirname "$HOMEBREW_BREW_FILE")")" && pwd -P)"
```

Comment:
style nit, may want to put this line in below with `HOMEBREW_REPOSITORY`, `HOMEBREW_LIBRARY`. It looks nice and more readable when putting them together,